stackdriver-debugger/src/com/google/cloud/tools/intellij/stackdriver/debugger/CloudDebugSettingsEditor.java

```diff
@@ -56,19 +57,14 @@ protected JComponent createEditor() {
 
   @Override
   protected void resetEditorFrom(@NotNull CloudDebugRunConfiguration runConfiguration) {
-    // TODO(ivanporty) CloudDebugRunConfiguration uses `cloudProjectName` for project ID.
-    String projectId = runConfiguration.getCloudProjectName();
+    String projectId = runConfiguration.getCloudProjectId();
+    // google username may be empty from previous revisions of configurations, should not be null.
+    String googleUsername = Optional.ofNullable(runConfiguration.getGoogleUsername()).orElse("");
     if (!Strings.isNullOrEmpty(projectId)) {
       settingsPanel.setSelectedCloudProject(
           CloudProject.create(
               // TODO(ivanporty) no project name in CloudDebugRunConfiguration.
-              projectId,
-              projectId,
-              null,
-              // TODO(ivanporty) no user name in CloudDebugRunConfiguration.
-              Optional.ofNullable(Services.getLoginService().getActiveUser())
-                  .map(CredentialedUser::getEmail)
-                  .orElse("")));
+              projectId, projectId, null, googleUsername));
```

Comment:
I know it was there before but could you please at a named param comment to the null param? (`.., null /* name*/, ..`)